abaco.go

```diff
@@ -45,25 +45,45 @@ type AbacoGroup struct {
 	lastSN     uint32
 }
 
+type AbacoUnwrapOptions struct {
+	Enable     bool // are we even unwrapping at all?
+	DropBits   bool // if false, return early and don't drop bits
+	ResetAfter int  // auto relock like number of samples to reset the phase unwrap offset back to 0 after
+	Bias       bool // should the unwrapping be biased?
+	PulseSign  int  // direction data will go when pulse arrives, used to calculate bias level
+}
+
+func (u AbacoUnwrapOptions) isvalid() error {
+	if u.Enable && !u.DropBits {
+		return fmt.Errorf("should not have both enable and DropBits true in AbacoUnwrapOpts")
+	}
+	return nil
+}
+
```

Comment:
If you remove the `DropBits` field like I propose (see other comments), then the parameters are more automatically self-consistent, and this entire test becomes unnecessary.